pages/common/nixpkgs-review.md

```diff
@@ -0,0 +1,29 @@
+# nixpkgs-review
+
+> Review pull requests in the NixOS packages repository (nixpkgs).
+> After a successful build, a `nix-shell` with all built packages will be started.
+> More information: <https://github.com/Mic92/nixpkgs-review#usage>.
+
+- Build changed packages in the specified PR:
```

Comment:
```suggestion
- Build changed packages in the specified pull request:
```
We could expand the abbreviation for non-technical users while leaving it as it is on the placeholders.